Assets/Scripts/Character/CharacterControl.cs

```diff
@@ -30,7 +30,9 @@ public bool FireWeapon(Vector3 targetPosition)
         rocketCooldownTimer = rocketCooldownDuration;
         var angle = Vector2.SignedAngle(Vector2.up, targetPosition - transform.position);
         var rotation = Quaternion.Euler(0.0f, 0.0f, angle);
-        Instantiate(rocketClass, transform.position, rotation);
+        var rocket = Instantiate(rocketClass, transform.position, rotation);
+        rocket.GetComponent<Rocket>().parentPlayer = gameObject;
+        Physics2D.IgnoreCollision(GetComponent<Collider2D>(), rocket.GetComponent<Collider2D>(), true);
```

Comment:
Maybe use variables to save the colliders instead? Then only need to call GetComponent() once at the start